qiskit_braket_plugin/providers/braket_provider.py

```diff
@@ -11,6 +15,9 @@ class AWSBraketProvider(ProviderV1):
     """AWSBraketProvider class for accessing AWS Braket backends."""
 
     def backends(self, name=None, **kwargs):
+
+        if(kwargs["local"]):
```

Comment:
```suggestion
        if kwargs.get("local"):
```
otherwise will fail if there is no `local` key in args